packages/modules/src/modules/headers/Header.tsx

```diff
@@ -56,6 +56,11 @@ const headingStyles = css`
 const Header: ReactComponent<HeaderRenderProps> = (props: HeaderRenderProps) => {
     const { heading, subheading, primaryCta, secondaryCta } = props.content;
 
+    const onClick = () => {
+        if (props.onCtaClick) {
+            props.onCtaClick();
+        }
```

Comment:
What do you think about using optional chaining here?

```suggestion
        props.onCtaClick?.();
```

Reply:
it is cleaner!